src/main/kotlin/app/revanced/patcher/Patcher.kt

```diff
@@ -48,70 +48,74 @@ class Patcher(private val options: PatcherOptions) {
 
     init {
         val extInputFile = ExtFile(options.inputFile)
```

Comment:
Use `.use` extension here instead of try-finally blocks.

Reply:
`ExtFile` does not implement `AutoCloseable`, so this will not work.